processor/workers.go

```diff
@@ -507,6 +510,12 @@ func CountStats(fileJob *FileJob) {
 			case SComment, SMulticomment, SMulticommentBlank:
 				fileJob.Comment++
 				currentState = resetState(currentState)
+				if MinifiedGenerated && !generatedComment && lineStart < 1000 {
```

Comment:
The problem I see with doing this here is that it will run over and over again for the first 1000 bytes which is going to impact performance. This logic should be pulled out above the core loop as it only needs to be done once.

That's going to be tricker though as you will need to handle the newline + bounds check yourself and ensure there is a comment. It might be acceptable to drop the comment check though as if you see the text generated or do not edit that might be good enough.

Reply:
Also I think it is better to remove `generated` from this check, and only keep `do not edit`, as `generated` alone produces false positives (even in `scc` repo).

Reply:
I was wondering about that. False positives are not ideal.

I think adding an option to supply the string to search for would help with this as well. So you invoke scc like so

```
scc --generated-string "do not edit",generated
```

or some such. This forces the user to implement it themselves avoiding the chance of induced false positives. 

Although a `-g` option or some such with some known generated types might still be useful as a base case.